src/views/NoteView.vue

```diff
@@ -2,19 +2,17 @@
   <div class="note-view">
     <div class="header-controls">
       <div class="note-info">
-        <div class="note-channel">channel:</div>
-        <div class="note-tags">tags:</div>
+        <div class="note-channel">channel:{{ note.channel }}</div>
+        <div class="note-tags">tags:{{ note.tag }}</div>
```

Comment:
こんな感じにラベル形式にすると良さそう
CSS の名前とか、基本的に `tag` と `tags` みたいに表記ゆれがないように修正してほしいです！

```suggestion
        <div class="note-channel">
          <span>channel: </span>
          <span class="note-channel">{{ note.channel }}</span>
        </div>
        <div class="note-tags">
          <span>tags: </span>
          <span class="note-tags">{{ note.tag }}</span>
        </div>
```